dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -2340,15 +2359,11 @@ def test_json_decode_after_update(
             ]
         )
 
-        deserialized.update_profile(df_structured)
+        with test_utils.mock_timeit():
+            deserialized.update_profile(df_structured)
+            expected_profile.update_profile(df_structured)
 
-        assert deserialized.total_samples == 5
-        assert deserialized._max_col_samples_used == 5
-        assert deserialized._min_col_samples_used == 5
-        assert deserialized.row_has_null_count == 2
-        assert deserialized.row_is_null_count == 1
-        assert deserialized._get_unique_row_ratio() == 0.80
-        assert deserialized.file_type == "<class 'pandas.core.frame.DataFrame'>"
+        test_utils.assert_profiles_equal(deserialized, expected_profile)
```

Comment:
now just validate that both update separately get same values

Reply:
very nice -- I like